src/api/DataSource/types.ts

```diff
@@ -37,6 +37,8 @@ export interface DataSourceFeatures {
   alerts: boolean;
   canDryRun: boolean;
   supportNativeDate: boolean;
+  searchEnabled: boolean;
+  isConsistent: boolean;
```

Comment:
here we are in "DataSourceFeatures". many things can be consistent in this scope. Name accordingly to the subject that it's "DataSourceFeatures".